subcommands/keys/ca_renewal_start.go

```diff
@@ -0,0 +1,92 @@
+package keys
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/foundriesio/fioctl/client"
+	"github.com/foundriesio/fioctl/subcommands"
+	"github.com/foundriesio/fioctl/x509"
+)
+
+func init() {
+	cmd := &cobra.Command{
+		Use:   "start <New PKI Directory> <Old PKI Directory>",
+		Short: "Start a root of trust renewal for your factory PKI",
+		Run:   doStartCaRenewal,
+		Args:  cobra.ExactArgs(2),
+		Long: `Start a root of trust renewal for your factory PKI.
+
+This command does a few things:.
+1. First, it generates a new root of trust for your factory.
+2. Second, it cross-signs a new root of trust using an old root of trust to prepare a CA renewal bundle.
+   This CA renewal bundle is compliant with the EST standard, necessary for a secure root CA update on devices.
+3. Finally, it uploads the CA renewal bundle to the backend API for validation and storage.
+
+A new root of trust needs to be stored in a separate directory from the previous root of trust.
+By the end of the renewal process, all necessary PKI pieces are migrated into this new directory.
+If you are using an HSM device - a new private key should be stored under a different label, possibly a new device.
+This extreme level of isolation is necessary to prevent an accidental rewrite of the old root of trust.
+
+Once this command completes successfully, a root of trust renewal process is started.`,
+	}
+	caRenewalCmd.AddCommand(cmd)
+}
+
+func doStartCaRenewal(cmd *cobra.Command, args []string) {
+	factory := viper.GetString("factory")
+	newCertsDir := args[0]
+	oldCertsDir := args[1]
+	if oldCertsDir == newCertsDir {
```

Comment:
probably should normalize these paths before doing this check so you don't get messed up by something like `start ./ $(pwd)`